examples/jest/__tests__/index.spec.js

```diff
@@ -19,12 +19,12 @@ describe("Dog's API", () => {
 
   const EXPECTED_BODY = [{dog: 1}]
 
-  beforeAll(() => provider.setup())
+  beforeAll(async () => await provider.setup())
```

Comment:
Why would we prefer the async/await dance here to the more concise and readable previous example?